cli/cmd/root.go

```diff
@@ -292,8 +292,8 @@ Use "{{.CommandPath}} [command] --help" for more information about a command.{{e
 `
 }
 
-// initConfig reads in config file and ENV variables if set
-func initConfig() {
+// InitConfig reads in config file and ENV variables if set
+func InitConfig() {
```

Comment:
why are we making this change? why make this function public?

Reply:
Forgot to revert this change. Was trying to use cache in the `init` of `cmd/component`. Had to call `InitConfig`. But ended up with using cache in the `RunE`.